vips/vip-442/bscmainnet.ts

```diff
@@ -0,0 +1,244 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+import { NORMAL_TIMELOCK } from "src/vip-framework";
+
+const { VTREASURY, RESILIENT_ORACLE, UNITROLLER, ACCESS_CONTROL_MANAGER, CHAINLINK_ORACLE } =
+  NETWORK_ADDRESSES.bscmainnet;
+export const VANGUARD_TREASURY = "0xf645a387180F5F74b968305dF81d54EB328d21ca";
+export const PROTOCOL_SHARE_RESERVE = "0xCa01D5A9A248a830E9D93231e791B1afFed7c446";
+export const SOL_CHAINLINK_FEED = "0x0E8a53DD9c13589df6382F13dA6B3Ec8F919B323";
+const SOL_MAX_STALE_PERIOD = 15 * 60; // 15 minutes
+const REDUCE_RESERVES_BLOCK_DELTA = "28800";
+
+const USDT = "0x55d398326f99059fF775485246999027B3197955";
+const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+const BTCB = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+const RISK_FUND_CONVERTER = "0xA5622D276CcbB8d9BBE3D1ffd1BB11a0032E53F0";
+const USDT_PRIME_CONVERTER = "0xD9f101AA67F3D72662609a2703387242452078C3";
+const USDC_PRIME_CONVERTER = "0xa758c9C215B6c4198F0a0e3FA46395Fa15Db691b";
+const BTCB_PRIME_CONVERTER = "0xE8CeAa79f082768f99266dFd208d665d2Dd18f53";
+const ETH_PRIME_CONVERTER = "0xca430B8A97Ea918fF634162acb0b731445B8195E";
+const XVS_VAULT_CONVERTER = "0xd5b9AE835F4C59272032B3B954417179573331E0";
+
+export const SOL_INCENTIVES = parseUnits("5000", 18);
```

Comment:
These funds are not for incentives. It's a refund. And it's not defined in SOL, but in USDT

Reply:
https://github.com/VenusProtocol/vips/commit/4997aaee6d2f4433ef205c49fb6761f9bd951cec